transformer_engine/common/include/transformer_engine/transformer_engine.h

```diff
@@ -82,6 +82,29 @@ enum NVTEScalingMode {
   NVTE_INVALID_SCALING
 };
 
+/*! \brief TE Quantization Params
+ *
+ * NVTEQuantizationParams is a struct storing a pointer
+ * to quantization params for fp8 quantization. It does not own the
+ * memory it points to.
+ */
+typedef void *NVTEQuantizationParams;
+
+/*! \brief Create a new TE quantization params.
+ *
+ *  \param[in] force_pow_2_scales Whether to force power of 2 scales
+ *  \param[in] amax_epsilon Small value to add to amax for numerical stability
+ *
+ *  \return A new TE quantization params.
+ */
+NVTEQuantizationParams nvte_create_quant_params(bool force_pow_2_scales, float amax_epsilon);
```

Comment:
This is not really a stable API yet since the function signature will change if we ever add new options. Long term, we'll want a workflow that's like:
```c++
NVTEQuantizationParams params = nvte_create_quantization_params();
bool pow_2_scales = true;
nvte_set_quantization_params(params, NVTE_QUANTIZATION_PARAMS_POW2SCALES, &pow_2_scales);
float fancyfeature123 = 456.f;
nvte_set_quantization_params(params, NVTE_QUANTIZATION_PARAMS_FANCYFEATURE123, &fancyfeature123);

...

nvte_destroy_quantization_params(params);
```
I think this is fine for now though.